### PR TITLE
Change hardcoded vector update to a for loop

### PR DESCRIPTION
**Pull request by jcvicelli**
iterating over the vector instead of hardcoding each item

### CODE REVIEW

File: src/references/exercise.rs [hunk 1]
```diff
@@ -25,9 +25,10 @@ fn magnitude(vector: &[f64; 3]) -> f64 {
 /// Change the magnitude of the vector to 1.0 without changing its direction.
 fn normalize(vector: &mut [f64; 3]) {
     let mag = magnitude(vector);
-    vector[0] /= mag;
-    vector[1] /= mag;
-    vector[2] /= mag;
+
```
Comment (mgeisler) [inline src/references/exercise.rs:28]: Nice! Let me remove the blank line to space on vertical space:
```suggestion
```

**Review by mgeisler — APPROVED** [review]
Thanks! I hope we've covered `for` loops at this point?

**Comment by djmitche** [comment]
The `course-content` binary is a good way to check that.

In this case, yes, `for` has been covered -- but only over ranges. Maybe we could compromise between idiomaticity and course flow with something like

```rust
for i in 0..3 {
  vector[i] /= mag;
}
```

On the other hand, this is in the solution to an exercise. If a student notices the for loop and asks, then the instructor can briefly explain it and circle back during the Iterators segment. And if nobody notices -- the syntax reads very naturally, so maybe students aren't surprised -- no problem!

**Comment by mgeisler** [comment]
> The `course-content` binary is a good way to check that.

Ah yes, I remember you created this a while ago :smile: As you can see, I haven't tried using it yet.

**Pull request merged**